src/Microsoft.AspNet.Authentication.Google/GoogleAuthenticationOptions.cs

```diff
@@ -28,5 +29,12 @@ public GoogleAuthenticationOptions()
         /// access_type. Set to 'offline' to request a refresh token.
         /// </summary>
         public string AccessType { get; set; }
+
+        /// <summary>
+        /// Defines whether access and refresh tokens should be stored in the
+        /// <see cref="ClaimsPrincipal"/> instance after a successful authentication.
+        /// This option is turned off by default to reduce the size of the final authentication cookie.
+        /// </summary>
+        public override bool SaveTokensAsClaims { get; set; } = false;
```

Comment:
I've thought too much about this PR already. :-)

I don't think the ability to change docs is compelling enough, especially when all that changes is the default. I also agree that the defaults are easier to understand in the constructor.